tests/suites/test_suite_ccm.data

```diff
@@ -13,10 +13,6 @@ CCM init #3 AES-224: bad key size
 depends_on:MBEDTLS_AES_C
 mbedtls_ccm_setkey:MBEDTLS_CIPHER_ID_AES:224:MBEDTLS_ERR_CCM_BAD_INPUT
 
-CCM init #4 BLOWFISH-128: bad block size
```

Comment:
Rather than remove this “bad block size” test case, we should keep it, but using DES instead of BLOWFISH.